internal/platform/scan/scan.go

```diff
@@ -259,11 +260,6 @@ func (s scanner) ScanEvents(ctx context.Context, fromBlock, toBlock *big.Int, co
 		return nil, fmt.Errorf("error instantiating ABI: %w", err)
 	}
 
-	collectionAbi, err := abi.JSON(strings.NewReader(contract.CollectionMetaData.ABI))
```

Comment:
Since this is not used anymore, I suggest deletiung `internal/platform/blockchain/contract/evolutioncollectionfactory.go`